scripts/filter_multimappers.py

```diff
@@ -70,7 +79,7 @@ def parse_arguments():
         "--nh",
         help=(
             "If set, the NH tag will be include in the alignment name after"
-            "and underscore. Default: %(default)s."
+            " and underscore. Default: %(default)s."
```

Comment:
"after and underscore"? Do you mean "an underscore"? Also, it should be "included" instead of "include"